core/tests/protractor_utils/forms.js

```diff
@@ -420,8 +443,10 @@ var MultiSelectEditor = function(elem) {
       expect(actualSelection).toEqual(expectedCurrentSelection);
 
       // Close the dropdown menu at the end.
-      await elem.element(by.css(
-        '.protractor-test-search-bar-dropdown-toggle')).click();
+      var testSearchBarDropdownToggle = elem.element(by.css(
+        '.protractor-test-search-bar-dropdown-toggle'));
+      await action.click(
+        'Test Search Bar Dropdown Toggle', testSearchBarDropdownToggle);
     }
   };
 };
```

Comment:
ditto: the many getText calls below. Please handle the getTagName and getAttribute calls below similarly

Reply:
done!